auth/rand/random_test.go

```diff
@@ -0,0 +1,30 @@
+package rand
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestToken(t *testing.T) {
+	for i := 1; i < 10; i++ {
+		assert.Len(t, Token(i), i)
+	}
```

Comment:
Is a loop in general a good thing for testing?
Yes, in this for loop you can clearly see how many times it runs and what is going to be asserted.
But in general this looks a little to generic for a test case to me.